src/views/SystemList/Table.tsx

```diff
@@ -277,28 +388,94 @@ export const Table = ({
         </tbody>
       </UswdsTable>
 
-      <div className="grid-row grid-gap grid-gap-lg">
-        <TablePagination
-          gotoPage={gotoPage}
-          previousPage={previousPage}
-          nextPage={nextPage}
-          canNextPage={canNextPage}
-          pageIndex={state.pageIndex}
-          pageOptions={pageOptions}
-          canPreviousPage={canPreviousPage}
-          pageCount={pageCount}
-          pageSize={state.pageSize}
-          setPageSize={setPageSize}
-          page={[]}
-          className="desktop:grid-col-fill"
-        />
-        <TablePageSize
-          className="desktop:grid-col-auto"
-          pageSize={state.pageSize}
-          setPageSize={setPageSize}
-        />
-      </div>
-    </>
+      {filteredSystems.length > 0 && (
+        <div className="grid-row grid-gap grid-gap-lg">
+          {filteredSystems.length > state.pageSize && (
+            <TablePagination
+              gotoPage={gotoPage}
+              previousPage={previousPage}
+              nextPage={nextPage}
+              canNextPage={canNextPage}
+              pageIndex={state.pageIndex}
+              pageOptions={pageOptions}
+              canPreviousPage={canPreviousPage}
+              pageCount={pageCount}
+              pageSize={state.pageSize}
+              setPageSize={setPageSize}
+              page={[]}
+              className="desktop:grid-col-fill"
+            />
+          )}
+          {filteredSystems.length > 5 && (
+            <TablePageSize
+              className="desktop:grid-col-auto"
+              pageSize={state.pageSize}
+              setPageSize={setPageSize}
+            />
+          )}
+        </div>
+      )}
+
+      {/* Alerts to show if there is no system/data */}
+      {filteredSystems.length === 0 &&
+        (tableType === 'my-systems' || isMySystems) && (
+          <Alert
+            type="info"
+            heading={t('systemTable.noMySystem.header')}
+            className="margin-top-5"
+          >
+            {isMySystems ? (
+              <Trans
+                i18nKey="systemProfile:systemTable.noMySystem.description"
+                components={{
+                  link1: (
```

Comment:
Check out how [this translation text sample](https://github.com/CMSgov/easi-app/blob/main/src/views/GovernanceTaskList/index.tsx#L138) might work for your purposes

I'm not sure about the TS issue

But we should at least try to use constants for emails